jtr.yml

```diff
@@ -0,0 +1,74 @@
+version: 1
```

Comment:
Is the only difference between this file and `jtr-isolate.yml` the retries and hard-retries? If so, those could be set via cli argumentst in the github workflow and we could just have `jtr.yml` for both local and browserstack.

Reply:
@timmywil PR updated, the latest run on my fork is still green: https://github.com/mgol/jquery-mousewheel/actions/runs/13866467470/job/38806483576